documentation/practices.md

```diff
@@ -80,5 +80,43 @@ Configuration properties for content URLs are:
+
+The React PWA Reference Storefront is pre-configured with a custom component for Indi integration. This component interacts with various widgets that Indi displays. New widgets may be added in the component as made available. For more information on Indi, see the company [website](https://indi.com/).
+
+The Indi component is a stand-alone component to fetch only the `indi-embed` libraries and display the elements corresponding to the libraries. These elements are passed to the component as a structured list.
+
+This component requires the following parameters:
+* `render`: The list of the elements to display from Indi. For example, `render={['carousel', 'brand', 'product']}`.
+Note:  Provide the correct configurations to render the element  correctly.
+* `configuration`: The structured object that consists of the configuration required for the Indi components. These configurations are defined in the `indi` parameter in `react-pwa-reference-storefront⁩/ep.config.json' file.
+* `keywords`: A string of the keywords for the product in which the Indi component is displayed.
+
+Static strings for the Indi component are localized in the Reference Storefront and are overridden whenever the Indi component is used. With this setting, the component displays the storefront in the locale that a customer chooses. The localized strings for the Indi component can be found in the corresponding language files within `src/localization`, for strings with the prefix `indi-`.
+
+The following are the configuration settings for the indi parameters in the `ep.config.json` file:
+
+| **Functionality**| **Parameter**| **Description**|
+|--|--|--|
+| **Integration**|`indi.enable`| The parameter that enables the component for the integration with Indi. For more information about the required configurations, see the company [website](https://indi.com/).|
+| **Theming**|`indi.carousel`| The configurations for the Indi carousel component.|
+| |`indi.carousel.apikey`| The _apikey_ that connects a carousel to Indi.|
+| |`indi.carousel.id`| The id used to connect the carousel to Indi.|
+| |`indi.carousel.size`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.theme`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.round_corners`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_title`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_views`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_likes`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_buzz`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.animate`| The parameter used to theme the carousel that displays the content from Indi.|
```

Comment:
Can we merge all rows with one `The parameter used to theme the carousel that displays the content from Indi.`